crates/ln-dlc-node/src/tests/dlc/dlc_setup_with_reconnects.rs

```diff
@@ -76,56 +69,47 @@ async fn reconnecting_during_dlc_channel_setup() {
     app.reconnect(coordinator.info).await.unwrap();
```

Comment:
🤔 Can we be sure here, that the coordinator hasn't processed the accept message before the reconnect?

Reply:
We can be sure in the sense that nothing happens unless we call `process_incoming_messages` (bad name because it also sends messages that are queued up) implicitly through `wait_until_dlc_channel_state` or `periodic_check` on the `SubChannelManager` implicitly through `sub_channel_manager_periodic_check`.

But we can certainly check that the message has not arrived yet, by asserting on the state of the app (should be `Offered`?) or maybe checking the coordinator's message queue manually.

Reply:
so isn't there a background task running automatically trying to process any incoming messages? If so, I think the asserting on the state should be fine. But, I think the `SubChannelState` of the app should be `Confirmed` after the coordinator processes the accept dlc channel offer?

btw. I noticed that in the tests we have the app and the coordinator roles inverted. But I guess that shouldn't have an implications.